### PR TITLE
Vulnerability fix | System.Text.Encodings.Web Version override

### PR DESCRIPTION
**Pull request by cheenamalhotra**
Change assessed by Component detection tool: [Build #31825](https://dev.azure.com/sqlclientdrivers-ci/sqlclient/_build/results?buildId=31825&view=results)

### CODE REVIEW

**Review by johnnypham — APPROVED** [review]
Does it not need to be added to the nuspec?

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj [hunk 1]
```diff
@@ -581,6 +581,9 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGitHubVersion)" PrivateAssets="All" />
+    <PackageReference Include="System.Text.Encodings.Web">
+      <Version>$(SystemTextEncodingsWebVersion)</Version>
+    </PackageReference>
```
Comment (DavoudEshtehari) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj:586]: NIT: Just prefering single line reference.
```suggestion
    <PackageReference Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWebVersion)" />
```

**Review by DavoudEshtehari — APPROVED** [review]

**Comment by cheenamalhotra** [comment]
> Does it not need to be added to the nuspec?

No, since the vulnerability arises from underlying library, we don't want to force override dependencies for all customers. We override this type of dependency for our product build only.

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj (hunk 1 above)
Comment (cheenamalhotra) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj:586]: I tried it before, this does not work in NetFx project.

**Review by JRahnama — APPROVED** [review]

**Pull request merged**